babel.config.js

```diff
@@ -7,9 +7,11 @@ const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 const modules = isCalypsoClient ? false : 'commonjs'; // only calypso should keep es6 modules
```

Comment:
I think we want to set `modules: false` also for `TARGET_BROWSER=true`. If we don't, tree shaking will not work. Only true server-side Node.js builds need CommonJS modules.